components/byu-faculty-profile/byu-faculty-profile-common.scss

```diff
@@ -39,7 +42,7 @@
 }
 
 @mixin cardContent() {
-  padding: 0px 16px 16px 16px;
+  padding: 8px 19px 16px 19px;
```

Comment:
The card title and the card content were lining up, so I added more padding to fix that.